site/docs/components/drawer/examples.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Drawer
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="drawer" exampleName="TopPositioned" >
+
+### Top Positioned
```

Comment:
should we add a generic descirption to each of these, including the prop name to change "position"